README.md

```diff
@@ -28,14 +29,81 @@ uv run orgwarden list-repos <org_url>
 ```
 
 ### Audit
-Runs [RepoAuditor](https://github.com/gt-sse-center/RepoAuditor) tooling. If the provided url points to a GitHub repository, RepoAuditor will run against said repository. If the provided url points to a GitHub organization, RepoAuditor will run against all public, non-forked repositories within said organization. The `gh-pat` option is a GitHub Personal Access Token (PAT) that is used by RepoAuditor for increased functionality.
+Runs [RepoAuditor](https://github.com/gt-sse-center/RepoAuditor) tooling. If the provided url points to a GitHub repository, RepoAuditor will run against said repository. If the provided url points to a GitHub organization, RepoAuditor will run against all public, non-forked repositories within said organization. The `gh-pat` option is a GitHub Personal Access Token (PAT) that RepoAuditor requires for full funcionality.
+
+**Note** - If a PAT is not provided, `audit` will run with limited functionality **and will always exit with a non-zero exit code.**
 
-*RepoAuditor currently only supports classic PATs.*
+*Due to an issue with the GitHub API, OrgWarden currently only supports Classic PATs.*
 ```bash
 uv run orgwarden audit <repo_or_org_url> --gh-pat [token]
 ```
 
 
+
+## Setting Up a Personal Access Token (PAT)
+A GitHub Personal Access Token (PAT) is required to make use of OrgWarden's full functionality. For information on creating Personal Access Tokens, see the [GitHub Docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+
+**Note** - Due to an issue with the GitHub API, your PAT must be a *Classic* token. *Fine-Grained* tokens are currently not supported.
+
+When creating your PAT, please ensure it has the following minimum permissions:
+
+repo:public_repo ✅
+
+<picture>
+    <img src="./public_repos_PAT.png" alt="GitHub Classic PAT Settings">
```

Comment:
Nit: I prefer to keep the root of a repository as clean as possible. Consider storing this image as `.github/images/public_repos_PAT.png`.